app/code/Magento/Customer/view/frontend/templates/widget/name.phtml

```diff
@@ -33,7 +33,6 @@ $suffix = $block->showSuffix();
     </label>
     <div class="control">
         <fieldset class="fieldset fieldset-fullname">
-        <div class="fields">
```

Comment:
@StevenGuapaBV This tag is closed in line 137 at the end of file. You can't delete it.

Reply:
I will double check the issue because I couldn't reproduce it.

Thanks for your contribution.